include/prereq-build.mk

```diff
@@ -188,7 +188,7 @@ $(eval $(call SetupHostCommand,python3,Please install Python >= 3.6, \
 
 $(eval $(call TestHostCommand,python3-distutils, \
 	Please install the Python3 distutils module, \
-	$(STAGING_DIR_HOST)/bin/python3 -c 'import distutils'))
+	$(STAGING_DIR_HOST)/bin/python3 -c 'from distutils import sysconfig'))
```

Comment:
`sysconfig` is deprecated since python 3.10, we should better look for something which is not deprecated like `from distutils import util`
https://docs.python.org/3.10/distutils/apiref.html#module-distutils.sysconfig
https://docs.python.org/3.10/distutils/apiref.html#module-distutils.util

Was `sysconfig` the function which some applications where missing?

Reply:
No it was a suggestion from https://github.com/openwrt/openwrt/issues/10325

All distutils functions are missing.

Reply:
@neheb I think this fix no longer works with Python 3.12 
`python3 -c 'from distutils import sysconfig'`
gives me
`
`

Reply:
The distutils module is now *provided* by setuptools.

Reply:
Note also that 3.12 is not actually released, so this is only a problem for people running custom testing builds, which is unlikely for OpenWrt.

Even you don't have the problem! You posted a warning, not an import error.

By the time 3.12 is released, Meson won't depend on distutils anymore for 3.12.

This won't actually help OpenWrt though, unless you drop python 3.6 support and upgrade Meson itself.

Reply:
@eli-schwartz  Okay, if OpenWrt is going to not depend on distutils  by the time of 3.12 then its not relevant. But to give some context, I get this when I try the `from distutils import sysconfig` same as `import distutils` when I run `./scripts/feeds update -a`:
```
Build dependency: Please install the Python3 distutils module
Prerequisite check failed. Use FORCE=1 to override.
```
I do have distutils module, so it must be that the DeprecationWarning is causing the check to fail.

Reply:
The check needs to be modified to only run on Python < 3.12

Reply:
This should've failed earlier. Python versions are usually whitelisted. Since a9e8eec221874e5840d659aed3f68850ff1d9982 3.12 now passed the check without any explicit whitelisting. The distutils check indeed only needs to adapted, but future versions shouldn't blindly be accepted for obvious reasons